cfn-resources/project-ip-access-list/mongodb-atlas-projectipaccesslist.json

```diff
@@ -95,35 +84,38 @@
       },
       "minItems": 1,
       "type": "array",
-      "uniqueItems": true
-    },
-    "ApiKeys": {
-      "$ref": "#/definitions/apiKeyDefinition"
-    },
-    "Id": {
-      "description": "The unique identifier for the Project API ip access list rules.",
-      "type": "string"
+      "uniqueItems": true,
+      "insertionOrder": false
     },
     "ProjectId": {
       "description": "Unique 24-hexadecimal digit string that identifies your project.",
       "type": "string"
     },
+    "ListOptions": {
+      "$ref": "#/definitions/listOptions"
+    },
     "TotalCount": {
       "description": "Number of documents returned in this response.",
       "type": "integer"
     },
-    "ListOptions": {
-      "$ref": "#/definitions/listOptions"
+    "Profile": {
+      "type": "string",
+      "description": "Profile used to provide credentials information, (a secret with the cfn/atlas/profile/{Profile}, is required), if not provided default is used",
+      "default": "default"
     }
   },
   "readOnlyProperties": [
-    "/properties/Id",
-    "/properties/TotalCount"
+    "/properties/AccessList/CIDRBlock",
+    "/properties/AccessList/IPAddress",
+    "/properties/AccessList/AwsSecurityGroup"
+  ],
+  "createOnlyProperties": [
+    "/properties/ProjectId",
+    "/properties/Profile"
```

Comment:
Why Profile is set to be createOnly? Here is the definition of creation:
createOnlyProperties
Resource properties can be specified by the user only during resource creation.

Reply:
I guess this is again a hack to retrieve the Profile by CFN during Read/List?

Reply:
Shouldn't the profile be a primary identifier?

Reply:
Sorry, maybe wrong question. I am a bit confused about these attributes.

Reply:
it makes sense. Thanks!

Reply:
Thanks for the explanation.